vm/src/obj/objint.rs

```diff
@@ -78,6 +78,23 @@ impl PyValue for PyInt {
     fn class(vm: &VirtualMachine) -> PyClassRef {
         vm.ctx.int_type()
     }
+
+    fn into_simple_object(self, vm: &VirtualMachine) -> PyObjectRef {
+        vm.ctx.new_bigint(&self.value)
```

Comment:
```suggestion
        vm.ctx.new_int(self.value)
```
I think `new_bigint` should probably be called `new_bigint_ref`, right? Since it's only for if you have a bigint reference that you might not want to clone if it fits into an int cache.

Reply:
Yes, exactly,